sender-stackdriver/pom.xml

```diff
@@ -42,17 +42,6 @@
     <dependency>
       <groupId>com.google.api.grpc</groupId>
       <artifactId>grpc-google-cloud-trace-v2</artifactId>
-      <version>${grpc-google-cloud-trace-v2.version}</version>
-      <exclusions>
```

Comment:
Was there something wrong with these exclusions? gRPC should always have compatibility even with stubs compiled against an older version - if they aren't that's quite surprising...

Reply:
it was more about the version skew. we can test if removing the exclusions doesnt whack the tree

Reply:
oh these are in the parent pom now.. that's what happened.